frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyNiceToHaveRequirements.yml

```diff
@@ -0,0 +1,21 @@
+name: Nice-to-have requirements
+id: niceToHaveRequirements
+question: Do you have any of the nice-to-have skills and experience?
+question_advice: |
+  Buyers will filter supplier applications based on the number of nice-to-have skills and experience they have.
+
+type: boolean_list
+number_of_items: 20
+optional: true
+
+depends:
```

Comment:
So it looks like all the brief response questions that are for all lots **do** define all three of the lots **apart from ** `essentialRequirements` and `niceToHaveRequirements` (which are new dynamic list questions we have added in).

I think we should go for a consistent approach so then the question is - is it beneficial to explicitly define the lots? If so, then we should add them for the ones that don't currently have them.